person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age? || parent_permission
+  end
```

Comment:
- [ ] Great job setting the of_age method to private but, its only the of_age method that should be private not the can_use_services so ill recommend either putting the can_use_services before the `private` keyword or adding `public` before the `can_use_services`